src/game/frontend/submenus/World/weather.hpp

```diff
@@ -0,0 +1,60 @@
+
+#include "core/commands/LoopedCommand.hpp"
+#include "game/features/Features.hpp"
+#include "game/rdr/Enums.hpp"
+#include "game/rdr/Natives.hpp"
+
+inline const char* WeatherTypes[]{
+
+    "BLIZZARD",
+    "CLOUDS",
+    "DRIZZLE",
+    "FOG",
+    "GROUNDBLIZZARD",
+    "HAIL",
+    "HIGHPRESSURE",
+    "HURRICANE",
+    "MISTY",
+    "OVERCAST",
+    "OVERCASTDARK",
+    "RAIN",
+    "SANDSTORM",
+    "SHOWER",
+    "SLEET",
+    "SNOW",
+    "SNOWLIGHT",
+    "SUNNY",
+    "THUNDER",
+    "THUNDERSTORM",
+    "WHITEOUT",
+};
+
+void ChangeWeather(const char* weather)
+{
+	MISC::_SET_OVERRIDE_WEATHER(MISC::GET_HASH_KEY(weather));
+}
+void ChangeTime(int H = 12, int M = 0, int S = 0, int transition = 0, bool freeze = false)
```

Comment:
turn into a command

Reply:
well not everything needs to be a command for example where you play the animations you could say to make the a command so yeah i think that is fine  i put every thing else as a command because i thought it was right but i don't think that the time changer has to be a command or change weather has to it can be one but at the end it doesn't

Reply:
I do agree here, because there are not methods to pass arguments to commands(minutes, hours, etc) so that makes sense.